Dockerfile.i386

```diff
@@ -1,61 +0,0 @@
-FROM balenalib/i386-debian:buster-build
```

Comment:
Why delete?

Reply:
why keep, do we need it? Thought it was forgotten.

Reply:
I don't think we need it, I added for people who may want it but I don't mind if you remove it.